### PR TITLE
docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md: add file

### PR DESCRIPTION
**Pull request by Pokisiekk**

### CODE REVIEW

**Comment by Pokisiekk** [comment]
@sulewskiprzemyslaw Commits should be verified now.

File: docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Supermicro X11 LGA1151 Series theory of operation
+
+## Remote power control using ipmitool
+
+To remote control supermicro you can use ipmitool.
+
+```bash
+ipmitool -H [bmc_ip] -U [bmc_username] -P [bmc_password] chassis power off
```
Comment (krystian-hebel) [inline docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md:8]: Where should I look for credentials?

Comment (macpijan) [reply docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md:8]: Could be a good idea to store them in snipeit, along with the other information, such as IP addresses?

Comment (Pokisiekk) [reply docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md:8]: @krystian-hebel I added credentiale on snipeit

File: docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+1. Edit the `/etc/default/grub` on your Linux-based OS file by adding:
+
+    ```bash
+    linux /vmlinuz ro console=ttyS1,115200n8 earlyprintk=serial,ttyS1,115200n8
+    ```
+
+1. Connect with BMC via ssh:
+
+    ```bash
+    ssh ADMIN@<IP>
```
Comment (macpijan) [inline docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md:34]: It should work via password authentication? It requires a key it appears. And documentation does not mention how to add one.

Comment (Pokisiekk) [reply docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md:41]: I prepared the documentation on a system with OpenSSH < 8.8, and it worked without any configuration.
https://github.com/Dasharo/docs/pull/585/commits/53c1c25d09798d99eef8d08b88cf1fea0e20f2c4 - I added the missing step

File: docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+Follow the steps below to set up a serial connection via ssh:
+
+1. Enable SOL/COM2 console in BIOS. To do that in BIOS go to the `Advanced`
+   section, then select `Serial Port Console Redirection` and set the
+   `SOL/COM2 Console Redirection` option to `Enabled` state.
+
+1. Edit the `/etc/default/grub` on your Linux-based OS file by adding:
+
+    ```bash
```
Comment (krystian-hebel) [inline docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md:27]:
```suggestion
    ```text
```

File: docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    cd system1/sol1
+    ```
+
+1. And start the serial console:
+
+    ```bash
+    start
+    ```
+
+If you want to run the SOL console again, repeat points 3-5.
```
Comment (krystian-hebel) [inline docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md:60]:
```suggestion
If you want to run the SOL console again, repeat points 4-6.
```

File: docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    > NOTE: If your Linux-based OS is QubesOS you should additionally remove the
+      `plymouth.ignore-serial-consoles` phase to enable password disk
+      authentication via SOL
+
+1. Add the following lines to the `/etc/ssh/ssh_config` file on your machine:
+
+    ```bash
+    HostKeyAlgorithms = +ssh-rsa
+    PubkeyAcceptedAlgorithms = +ssh-rsa
+    ```
```
Comment (krystian-hebel) [inline docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md:40]: Adding such settings to global configuration is asking for problems...

```suggestion
1. Add the following lines to the `~/.ssh/config` file on your machine:

    ```text
    Host <IP>
    	HostKeyAlgorithms = +ssh-rsa
    	PubkeyAcceptedAlgorithms = +ssh-rsa
    ```
```

File: docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md (hunk 2 above)
Comment (Pokisiekk) [reply docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md:46]: @macpijan ping

Comment (miczyg1) [reply docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md:46]: @krystian-hebel any ideas?

Comment (macpijan) [reply docs/transparent-validation/supermicro_x11_lga1151_series/theory-of-operation.md:46]: The updated documentation worked for me.

**Review by macpijan — APPROVED** [review]

**Pull request merged**